CHANGELOG

```diff
@@ -1,4 +1,14 @@
 # Changelog
+## [0.2.2] - 2019-02-03
+### New Features
+ - Team interface support
+
+### Bug Fixes
+ - Fix memory leak of NM.Client
```

Comment:
I think this was a file descriptor leak, how about `Fix file descriptor leak` - the NM.Client part is an internal detail

Reply:
Sure, done thanks!